src/components/operations/Operations.jsx

```diff
@@ -186,7 +187,7 @@ export default class PendingOperations extends PureComponent {
 
 function mapStateToProps (state) {
   const operations = state.get('operations')
-  const network = state.get('network')
+  const network = state.get(DUCK_NETWORK)
```

Comment:
For the line 190.

Please create appropriate constant for the 'operations' duck: DUCK_OPERATIONS.
Multiple usage of 'operations' string accross the project.